app/jobs/create_source_file_job.rb

```diff
@@ -33,11 +33,15 @@ def maybe_link_to_original_source_file(source_file, linkable_word_files)
     end
 
     if original_source_file.present?
-      source_file.update!(original_source_file:)
+      source_file.update!(
+        original_source_file:,
+        assignee: original_source_file.assignee
```

Comment:
@littleforest is it guaranteed that every source file has an assignee?

Reply:
No, but then it will just assign `nil`, which is fine.